templates/js/copilot-plugin-from-scratch/.vscode/settings.json

```diff
@@ -12,9 +12,5 @@
         "obj": true,
```

Comment:
`files.exclude` is for what?

Reply:
I copied it from the template of azure functions, if we don't need it for TTK, I will remove it.